community.md

```diff
@@ -0,0 +1,143 @@
+        </a>
+        <a href="https://trinodb.slack.com/archives/CFLB9AMBN" class="slack-channel slack-pink-gradient">
+          <i class="fa fa-home fa-lg"></i>
+          #general
+        </a>
+        <a href="https://trinodb.slack.com/archives/CGB0QHWSW" class="slack-channel slack-orange-gradient">
+          <i class="fa fa-tools fa-lg"></i>
+          #troubleshooting
+        </a>
+      </div>
+      <div class="slack-channels-bottom d-flex">
+        <a href="https://trinodb.slack.com/archives/CP1MUNEUX" class="slack-channel slack-orange-gradient">
+          <i class="fa fa-code fa-lg"></i>
+          #dev
+        </a>
+        <a href="https://trinodb.slack.com/archives/CFSDUNJ4W" class="slack-channel slack-pink-gradient">
+          <i class="fa fa-dharmachakra fa-lg"></i>
+          #kubernetes
+        </a>
+        <a href="https://trinodb.slack.com/archives/CJ6UC075E" class="slack-channel slack-blue-gradient">
+          <i class="fa fa-server fa-lg"></i>
+          #iceberg
+        </a>
+        <a href="https://trinodb.slack.com/archives/CFQAMGRQE" class="slack-channel slack-pinkorange-gradient">
+          <img src="../assets/images/graphics/community-announcements.svg">
+          <i class="fa fa-bullhorn fa-lg"></i>
+          #community-announcements
+        </a>
+      </div>
+    </div>
+    <div class="community-content-section">
+      <h2>Events that don’t suck</h2>
+      <div class="community-content-container">
+        <a href="./broadcast" class="community-broadcast-block">
+          <div class="community-broadcast-image-wrapper">
+            <img src="../assets/images/community/tcb.png" />
+            <img src="../assets/images/community/tcb.gif" />
+          </div>
+        </a>
+        <div class="events-thirds-container">
+          <a href="https://www.meetup.com/topics/trino/" class="event-third-block"></a>
+          <a href="https://twitter.com/trinodb" class="event-third-block">
+            <div>Follow us<h2>@trinodb</h2></div>
+            <img src="../assets/images/community/twitter.png" />
+          </a>
+          <a href="https://www.youtube.com/c/trinodb" class="event-third-block">
+            <div>Catch up on what's happening in the Trino community</div>
+            <img src="../assets/images/community/youtube.png" />
+          </a>
+        </div>
+      </div>
+    </div>
+    <div class="community-content-section">
+      <h2>Contribute</h2>
+      <div class="community-content-container contribute-content-container">
+        <div class="contribute-text-container">
+          <div class="contribute-github-button d-flex flex-row">
+            <a class="github-button" href="https://github.com/trinodb" data-size="large" aria-label="Follow @trinodb on GitHub">Follow @trinodb</a>
+            <a class="github-button" href="https://github.com/trinodb/trino" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star trinodb/trino on GitHub">Star</a>
```

Comment:
Maybe text needs to mention Star Trino on GitHub?

Reply:
This renders as a GitHub button with a Star icon.